src/OrbitGl/TextRenderer.cpp

```diff
@@ -25,12 +25,26 @@
 #include "OrbitBase/ExecutablePath.h"
 #include "OrbitBase/Logging.h"
 
+namespace {
+
 struct vertex_t {
   float x, y, z;     // position
   float s, t;        // texture
   float r, g, b, a;  // color
 };
 
+int GetStringLineCount(const char* string) {
```

Comment:
Question, making it a std::string and counting with count is too much? std::count (string.begin(), string.end(), '\n')?

Reply:
_Probably_ not, but I'd prefer to stick with this version for now - `TextRenderer::AddText` doesn't perform a conversion to `std::string`, so it's definitely some additional overhead, and I don't feel the current version has to be rewritten for clarity, even though I like your suggestion better.

If anything it would probably be worth to just change the API of `TextRenderer::AddText` to receive a `const std::string&` instead of `const char*`, we're already calling it from a `std::string` most of the times, but I don't want to do this in this PR.